packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -214,6 +214,8 @@ $groupPubPackages
     // TODO(dnys1): Enable E2E runs for Dart packages
     final needsE2ETest = package.flavor == PackageFlavor.flutter &&
         package.integrationTestDirectory != null;
+    final hasGoldens = package.flavor == PackageFlavor.flutter &&
+        package.goldensTestDirectory != null;
```

Comment:
An alternative to checking for a directory would be to look for a dev dep of `golden_toolkit`. It isn't required to run golden tests, but I think we would be likely to use it in any future packages that have golden tests. Looking for a ui directory is fine too. If this is still in place when future UI components are built, we will just have to confirm to that dir.

Reply:
Understood, this for sure could be different. Since we don't have any other UI packages or other established conventions, we can readdress this down the road.